apps/web/src/pages/api/infinity/pools.ts

```diff
@@ -199,3 +211,27 @@ async function fetchAllPools({
 
   return allResults
 }
+
+export const poolTvlMap = async (
+  protocols: ('v2' | 'v3' | 'infinityBin' | 'infinityCl' | 'stable')[],
+  chain: APIChain,
+) => {
+  try {
+    const remotePools = await fetchAllPools({
+      baseUrl: 'https://explorer.pancakeswap.com/api/cached/pools/list',
```

Comment:
we have a new endpoint where you can get all the tvl references
https://explorer.pancakeswap.com/api/doc#tag/pools/GET/cached/pools/tvl-refs